rust/zypp-agama/src/lib.rs

```diff
@@ -307,6 +322,34 @@ impl Zypp {
         }
     }
 
+    pub fn disable_repository(&self, alias: &str) -> ZyppResult<()> {
+        unsafe {
+            let mut status: Status = Status::default();
+            let status_ptr = &mut status as *mut _ as *mut Status;
```

Comment:
```suggestion
            let status_ptr = &mut status as *mut _;
```

Clippy said shorter incantations are fine :)